modules/gridNMBidAdapter.js

```diff
@@ -139,11 +163,14 @@ export const spec = {
             height: serverBid.h,
             creativeId: serverBid.auid || bid.bidderRequestId,
             currency: 'USD',
-            netRevenue: false,
+            netRevenue: true,
             ttl: TIME_TO_LIVE,
             dealId: serverBid.dealid,
             vastXml: serverBid.adm,
             mediaType: VIDEO,
+            meta: {
+              advertiserDomains: serverBid && serverBid.adomain ? serverBid.adomain : []
```

Comment:
Is the serverBid check here necessary given line 148? This can simplify to `serverBid.adomain || []`